docs-ui/components/featureDisabled.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import {storiesOf} from '@storybook/react';
+import {withInfo} from '@storybook/addon-info';
+import FeatureDisabled from 'app/components/acl/featureDisabled';
+
+storiesOf('UI|FeatureDisabled', module)
+  .add(
+    'basic style',
+    withInfo('A disabled feature component')(() => (
+      <FeatureDisabled name="Example Feature" feature="organizaton:example-feature" />
+    ))
+  )
+  .add(
+    'alert style',
+    withInfo('A disabled feature wrapped in an alert')(() => (
+      <FeatureDisabled
+        name="Example Feature"
+        feature="organizaton:example-feature"
```

Comment:
Yeah I'm tempted to say we should remove that "feature" since it could absolutely cause ambiguity and confusion.

I'll follow up with that at some point.